cogs/events.py

```diff
@@ -3,22 +3,21 @@
 It contains the following commands:
 - list_events: Lists the events associated with this guild.
 - delete_event: Deletes an event from the guild.
+- on_ready: Handles the on_ready event. This event is triggered when the bot has successfully connected.
```

Comment:
Think it would be good to put the event names in "quotes". Like `Handles the "on_ready" event.`